{"uid":"84f369ba8c3ae14e","name":"shouldShowErrorWhenUsingNonexistentAccount()","fullName":"org.saucedemo.tests.authentication.AuthenticationTest.shouldShowErrorWhenUsingNonexistentAccount","historyId":"4213748117e1819ba4bc7318b2dff0e6","time":{"start":1755863927421,"stop":1755863928229,"duration":808},"description":"Validates that user is unable to log in using a username of an account that doesn't exist in the application's database, and is being shown an error with the expected description.","descriptionHtml":"<p>Validates that user is unable to log in using a username of an account that doesn't exist in the application's database, and is being shown an error with the expected description.</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:org.saucedemo.tests.authentication.AuthenticationTest]/[method:shouldShowErrorWhenUsingNonexistentAccount()]"},{"name":"epic","value":"Authentication"},{"name":"feature","value":"Login Functionality"},{"name":"story","value":"User tries to log in using an account that doesn't exist in the database."},{"name":"host","value":"pkrvmqc4gcfdwos"},{"name":"thread","value":"2531@pkrvmqc4gcfdwos.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"org.saucedemo.tests.authentication.AuthenticationTest"},{"name":"testClass","value":"org.saucedemo.tests.authentication.AuthenticationTest"},{"name":"testMethod","value":"shouldShowErrorWhenUsingNonexistentAccount"},{"name":"suite","value":"org.saucedemo.tests.authentication.AuthenticationTest"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[],"tags":[]},"source":"84f369ba8c3ae14e.json","parameterValues":[]}